### ISSUE TITLE
ROUGE Scores do not match with the ones mentioned in the Paper

### ISSUE THREAD

**Issue by vikasTmz** [PrekshaNema25/diversity_based_attention#3]
Trained on the hyperparam given in config.txt and on the same dataset:
```
rouge-1:  26.888950791266154
rouge-2:  1.7499898144222332
rouge-l:  20.842653462853296
```

**Comment by genzen2103** [comment]
Hi, Trained on default hyper parameters in config. Used rouge python package for computing scores
Below are results:
Test Loss:6.344333601
rouge-2
p:1.5042511445389144
r:4.4689119170984455
f:2.2508559272005666
rouge-1
p:8.380173735309146
r:26.472962066182404
f:12.730447917294093
rouge-l
p:9.86910255202938
r:18.865886588658867
f:10.995739923842024

As you can see they dont match the optimal ones in paper. Please let me know if any changes are needed in the code
